src/Hazelcast.Net.Tests/Configuration/HazelcastOptionsTests.cs

```diff
@@ -489,6 +590,11 @@ public void Dispose()
             { }
         }
 
+        public class DummyBinder
+        {
+            public int Count { get; set; }
+        }
+
```

Comment:
Is this used somewhere?

Reply:
Fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/679/commits/b18db521cd42644ac6a615e314f9b18525b6bfa5